src/components/reset/ResetPage.tsx

```diff
@@ -0,0 +1,251 @@
+            ))}
+            <Button
+              type="button"
+              onClick={() =>
+                appendScoringComponent({
+                  friendlyName: "",
+                  isSidepot: false,
+                  id: generateId(),
+                })
+              }
+            >
+              Add Score Component
+            </Button>
+          </div>
+          <div className="flex w-1/3 flex-col gap-2">
+            <Label htmlFor="scoringSidepots">Scoring Sidepots (JSON): </Label>
+            {scoringSidepots.map((field, index) => (
+              <div key={field?.id} className="flex flex-row gap-2">
+                <Input
+                  required
+                  id="scoringSidepots"
+                  {...register(`scoringSidepots.${index}.friendlyName`)}
+                  placeholder="Usability, Creativity, etc."
+                />
+                <Button
+                  variation="primary"
+                  colorTheme="error"
+                  type="button"
+                  onClick={() => removeScoringSidepot(index)}
+                >
+                  X
+                </Button>
+              </div>
+            ))}
+            <Button
+              type="button"
+              onClick={() =>
+                appendScoringSidepot({
+                  friendlyName: "",
+                  isSidepot: true,
+                  id: generateId(),
+                })
+              }
+            >
+              Add Scoring Sidepot
+            </Button>
+          </div>
+        </div>
+        <div className="flex flex-row">
+          {" "}
+          <div className="mr-24 flex w-1/3 flex-col gap-2">
+            <div className="flex flex-col gap-2">
+              <Label htmlFor="startDate">Start Date:</Label>
+              <Input
+                required
+                type="date"
+                id="startDate"
+                placeholder="Last Name"
+                {...register("startDate")}
+              />
+            </div>
+            <div className="flex flex-col gap-2">
+              <Label htmlFor="endDate">End Date: </Label>
+              <Input
+                required
+                type="date"
+                id="endDate"
+                placeholder="Last Name"
+                {...register("endDate")}
+              />
+            </div>
+          </div>{" "}
+          <div className="flex w-1/3 flex-col gap-2">
+            <Label>Reset Fields</Label>
+            <CheckboxField label="Reset users: " {...register("resetUsers")} />
+            <CheckboxField label="Reset Teams: " {...register("resetTeams")} />
+            <CheckboxField label="Reset Rooms: " {...register("resetRooms")} />
+            <CheckboxField
+              label="Reset Scores: "
+              {...register("resetScores")}
+            />
+          </div>
+        </div>
+
+        <div className="flex flex-col gap-2">
+          <Label htmlFor="safetyCheck">Safety Check: </Label>
```

Comment:
Rename Label to: **Enter "deletehackathon" to confirm**

```suggestion
          <Label htmlFor="safetyCheck">Enter "deletehackathon" to confirm </Label>
```